src/core/lib/security/context/security_context.h

```diff
@@ -21,6 +21,7 @@
 
 #include <grpc/support/port_platform.h>
 
+#include "src/core/lib/gpr/arena.h"
```

Comment:
One more nit: rather than include the arena in the header, can you forward declare it here, then only include it in the .cc file

Reply:
Note google C++ style guide prefers #include to forward declaration :)

Reply:
Welp, I didn't believe it, and I looked it up... 

https://google.github.io/styleguide/cppguide.html#Forward_Declarations

I stand corrected. Hope, I have literally given you the opposite of good advice in this PR